arm/Microsoft.Web/sites/deploy.bicep

```diff
@@ -38,10 +38,7 @@ param functionsWorkerRuntime string = ''
 param functionsExtensionVersion string = '~3'
 
 @description('Optional. The resource ID of the app service plan to use for the site. If not provided, the appServicePlanObject is used to create a new plan.')
-param appServicePlanId string = ''
-
-@description('Optional. Required if no appServicePlanId is provided to deploy a new app service plan.')
-param appServicePlanObject object = {}
+param serverFarmResourceId string = ''
```

Comment:
Is this description still true?

Reply:
Good catch. It's not - updated the description accordingly